reddit/scanner.go

```diff
@@ -60,4 +60,4 @@ func (s *scanner) ListingWithParams(path string, params map[string]string) (
 		reaperParams[key] = value
 	}
 	return s.r.reap(path, reaperParams)
-}
\ No newline at end of file
+}
```

Comment:
I think this is an editor artifact.

Reply:
Actually, other files have a line feed but not this one